empire/core/empire.py

```diff
@@ -79,7 +82,8 @@ def run_empire(name, tab_file_path: Path, result_file_path: Path, scenario_data_
 
     #Spatial sets
     model.Node = Set(ordered=True) #n
-    model.OffshoreNode = Set(ordered=True, within=model.Node) #n
+    if north_sea:
+        model.OffshoreNode = Set(ordered=True, within=model.Node) #n

```

Comment:
I det aggregerte datasettet er det ingen offshore-noder, legger derfor inn en sjekk på north_sea for å unngå errors